apps/web/src/server/applications/case/documentation-metadata/__tests__/documentation-metadata.test.js

```diff
@@ -361,8 +361,8 @@ describe('Edit applications documentation metadata', () => {
 
 			it('should return an error if the date is in the future', async () => {
 				const response = await request.post(`${baseUrl}/receipt-date`).send({
-					'dateCreated.day': '1',
```

Comment:
I had to apply the new format constraint (DD/MM/YYYY format for dates) to every other page that contains a date input (i.e. document metadata and the key dates page in the 'create new case' journey).

Tests and code are updated accordingly.

Reply:
Also, I had to separate the validator that checks whether the date exists from the validator the checks the format. This again applies to every page with a date input.